news/4.31/platform.html

```diff
@@ -321,8 +321,27 @@ <h2>General Updates </h2>
 	 - org.eclipse.ui.intro.quicklinks 1.2.300
 	   - because it Require-Bundle: org.eclipse.ui.workbench; bundle-version="3.108.0"
         </pre>
+        </p>
       </td>
   </tr>
+
+  <tr id="windows-defender-auto-config">
+    <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1453 -->
+    <td class="title">Windows Defender auto configuration</td>
+    <td class="content">
+      <p>
+        On Mirosoft Windows 10 or later the Microsoft Defender can significantly slow-down the startup and overall performance of Eclipse-based applications.
+        The Eclipse IDE can now detect if Microsoft Defender is active, inform you about that and if wanted can exclude itself from future Defender scans.
+      </p>
+      <p><img src="images/windows-defender-autofix_startup-check.png" alt="Windows Defender autofix startup check"/></p>
+      <p>
+        On Windows the <code>General &#62; Startup and Shutdown</code> preference page offers further information 
```

Comment:
We don't address preference pages consistently.
Here we use <code></code> on entries above we use <em>Appearance</em> (even without a full path to that page.

Which way should be choose?

Reply:
I have no all unified all preference references I found to use `->`.
In the end I think there is not much difference between `->` and `>`, it should just be used uniformly.